projects/transformers/export_model.py

```diff
@@ -31,13 +31,16 @@
 
 from transformers import AutoModelForMaskedLM
 
+# Import models. This will update Transformer's model mappings so that custom models can
+# be loaded via AutoModelForMaskedLM.
```

Comment:
Not sure I got this magic. Can you please explain it how this works in our next call?

Reply:
Sure thing. The basic idea is that our custom sparse models get registered in the appropriate dictionaries so that functions like `AutoModelForMaskedLM` can use them. This happens as soon as they are defined thanks to `register_bert_model`. And they get defined as soon as you import the `models` module.

Reply:
If it is changing a constant in the global namespace, does that mean it is dependent on the order it is imported? If we change the import order, does it still work as expected?

Reply:
@mvacaporale just making sure you saw this question, that is still not clear to me.